tst/testbugfix/2018-06-28-grplatt.tst

```diff
@@ -0,0 +1,7 @@
+# #2586
+
+gap> g:=TransitiveGroup(6,14);;
+gap> cl:=ConjugacyClassesSubgroups(g);;
+gap> ForAll(cl,x->IsSubset(g,Representative(x)));
+true
+
```

Comment:
This test file actually fails the test due to this extra new line at the end. Removing it should be enough to fix the test.